src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -42,8 +42,10 @@ export function useHeatmapSize<T>(
   return [
     wrapperRef,
     {
-      width: meshWidth + leftAxisWidth,
-      height: meshHeight + bottomAxisHeight,
+      width: meshWidth,
+      height: meshHeight,
+      paddingBottom: bottomAxisHeight,
+      paddingLeft: leftAxisWidth,
```

Comment:
This is the most important bit: the width and height of the `heatmap` element in `HeatmapVis` now match the width and height of the canvas and of the mesh. It uses padding to leave space for the axis. Not that the CSS adds `box-sizing: content-box`, so the total width of the `heatmap` element ends up being `width + leftAxisWidth`.

Reply:
I moved the `content-box` style here even though it's static, so the code is more self-documenting.